addons/MetroidvaniaSystem/Database/MapEditor.tscn

```diff
@@ -627,6 +627,19 @@ func create_scene_confirm() -> void:
 	EditorInterface.get_resource_filesystem().scan()
 	EditorInterface.open_scene_from_path(scene_path)
 	on_map_selected(scene_path)
+
+func can_drop_data(at_position: Vector2, data) -> bool:
+	if highlighted_room.is_empty():
+		return false
+	if data.get(\"type\", \"\") != \"files\" or len(data.files) != 1:
+		return false
+	if not data.files[0].begins_with(MetSys.settings.map_root_folder):
+		return false
+	return true
```

Comment:
It's unlikely, but the filter can accept e.g. image files from map folder. You should add an extension check too (`tscn`, `scn`).